doc/userguide_data_cleaning.rst

```diff
@@ -0,0 +1,72 @@
+.. _userguide_data_cleaning:
```

Comment:
I think that this label should be just above the corresponding title, else the pointer to the title is broken